### ISSUE TITLE
Error duplicating products if system has multi-site

### ISSUE THREAD

**Issue by watarutmnh** [craftcms/commerce#1491]
**Description**
An error occurs when duplicating a product if the system has multiple sites running.
Here is the error detail:
```
HTTP 500 – Internal Server Error – yii\web\ServerErrorHttpException
An error occurred when duplicating the product.
↵
Caused by: Integrity constraint violation – yii\db\IntegrityException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'dateCreated' cannot be null
The SQL being executed was: UPDATE `commerce_products` SET `defaultPrice`='0', `defaultHeight`='0', `defaultLength`='0', `defaultWidth`='0', `defaultWeight`='0', `dateCreated`=NULL, `dateUpdated`='2020-06-08 08:17:17' WHERE `id`=29
in /project-folder/vendor/yiisoft/yii2/db/Schema.php at line 674

Error Info: Array
(
    [0] => 23000
    [1] => 1048
    [2] => Column 'dateCreated' cannot be null
)
↵
Caused by: PDOException
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'dateCreated' cannot be null
in /project-folder/vendor/yiisoft/yii2/db/Command.php at line 1293
```

**To Reproduce**
1. Install commerce plugin
2. Add an additional site
3. Try to duplicate initially created product "A New Toga"
4. An error occurs

**Additional info**
- Craft version: 3.4.22.1
- PHP version: 7.3.14
- Database driver & version: MySQL 5.7.29
- Craft Commerce	3.1.7

**Comment by engram-design** [comment]
+1 for this issue as well.

**Comment by lukeholder** [comment]
Working on it as we speak.

**Comment by brandonkelly** [comment]
We just released Commerce 3.1.8 with this fix.
